libbebop/Makefile

```diff
@@ -6,9 +6,7 @@ include ../common/flags.mk
 CXXFLAGS+=-Wno-deprecated-declarations
 
 # AR SDK
-ifndef ARSDK_ROOT
-	$(error Environment variable ARSDK_ROOT must be defined)
-endif
+ARSDK_ROOT ?= /opt/arsdk3
```

Comment:
I'm a little confused by this - I don't think this is a very standard-looking default and I never actually saw it failing on my machine

Reply:
Yeah it probably isn't a very standard default... It turns out it was my machine which was complaining about ARSDK_ROOT not being set -- I thought it was yours by mistake. The weird thing is that I have ARSDK_ROOT set in ~jenkins/.profile and in my /etc/profile.d folder and neither of those things seem to help. Did you set ARSDK_ROOT in the Jenkins settings for your machine or something?

Reply:
Ahh yes - the happy hours I spent attempting to hack this for GeNN...

Luckily newer Jenki' have made this easier and there's now a mechanism to inject environment variables - which I have used to set ARSDK_ROOT for your machine. Could you remove the default and check it builds?